src/modules/form/util/formUtil.tsx

```diff
@@ -1375,11 +1388,12 @@ export const getDependentItemsDisabledStatus = ({
 }: GetDependentItemsDisabledStatus) => {
```

Comment:
Poking around at this because I'm looking at a difference with the DisabilityTable. Maybe I'm missing something obvious but I'm not seeing that `getDependentItemsDisabledStatus` is used anymore? (Or maybe it already wasn't?)

Reply:
good catch! It was used previously but I've removed it